Makefile

```diff
@@ -629,6 +629,7 @@ k3d/retest:
 			LAGOON_FEATURE_FLAG_DEFAULT_ISOLATION_NETWORK_POLICY=enabled \
 			USE_CALICO_CNI=false \
 			LAGOON_FEATURE_FLAG_DEFAULT_ROOTLESS_WORKLOAD=enabled \
+			CLEAR_API_DATA=true \
```

Comment:
I'd like to see this changed to set `CLEAR_API_DATA` as a make var with the default `true` similar to other options (https://github.com/uselagoon/lagoon/blob/testing/fix_dwp_rerun/Makefile#L57)

Then update this to use the make var, this way its possible to override it in the command line to `false` if required.
```
CLEAR_API_DATA=$(CLEAR_API_DATA)
```